lib/daru/io/importers/excel.rb

```diff
@@ -45,22 +55,26 @@ class Excel < Base
         #   #    3        4    Franz      nil    Paris      nil
         #   #    4        5   George      5.5     Tome    a,b,c
         #   #    5        6  Fernand      nil      nil      nil
-        def initialize(path, worksheet_id: 0)
+        def initialize(path, sheet: 0, headers: true)
```

Comment:
Any specific reason for changing to `sheet`? It will break backwards compatibility.